introduction.md

```diff
@@ -1 +1,4 @@
 This is my intro, check it out. The path I am following is Ruby.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?